pages/[type]/[id]/changelog.vue

```diff
@@ -235,16 +239,14 @@ function switchPage(page) {
     }
 
     .download {
+      position: absolute;
```

Comment:
This is not the solution I'd use for this, because there will be overlapping on long titles. Instead, I'd use a `margin-left: auto` to push it to the far right.